components/brave_wallet_ui/stories/locale.ts

```diff
@@ -885,5 +885,11 @@ provideStrings({
   braveWalletNftPinningErrorTooltipReasonTwo: 'Internal IPFS node problems',
   braveWalletNftPinningErrorTooltipReasonThree: 'Not enough space on local node',
   braveWalletImportNftModalTitle: 'Import NFT',
-  braveWalletEditNftModalTitle: 'Edit NFT'
+  braveWalletEditNftModalTitle: 'Edit NFT',
+  
+  // NFT auto discovery modal
+  braveWalletEnableNftAutoDiscoveryModalHeader: 'Want your NFTs displayed automatically?',
+  braveWalletEnableNftAutoDiscoveryModalDescription: 'Most wallets, including Brave Wallet, need a third-party service to display NFTs automatically. Brave works with $1SimpleHash$2 to provide this capability by $3sharing your public address.$4',
+  braveWalletEnableNftAutoDiscoveryModalConfirm: 'Yes, proceed',
+  braveWalletEnableNftAutoDiscoveryModalCancel: 'No thanks, I\’ll do it manually'
```

Comment:
cc @PrivacyMatters for text review

Reply:
does this need to link to the simplehash privacy policy or ToU or something like that?

Reply:
@diracdeltas I have opened a sec/privacy issue here: https://github.com/brave/security/issues/1248

Reply:
The text looks as [per](https://user-images.githubusercontent.com/48117473/231673666-e11ca08e-f2e8-44d3-8780-9d57be8787cf.png) fine and good to see individuals being given choice.